package.json

```diff
@@ -48,7 +45,7 @@
     "postversion": "git push --set-upstream ${GIT_REMOTE:-origin} release/v$npm_package_version"
   },
   "config": {
-    "k8sProxyVersion": "0.1.2",
+    "k8sProxyVersion": "0.1.5",
```

Comment:
This change probably should not be part of this PR?

Reply:
I can make that a separate PR yes.

Reply:
PR is now up https://github.com/lensapp/lens/pull/4946

Reply:
Idk why this change is still visible here? Maybe rebase with master?

Reply:
Yes I should rebase with master.